public/index.html.erb

```diff
@@ -5,24 +5,137 @@
   <script src="/scripts/url.min.js"></script>
   <script src="/scripts/register.js"></script>
   <script src="/scripts/stats.js"></script>
+  <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
+   <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
+   <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700,400italic,300italic,300,600italic,600,700italic,800,800italic' rel='stylesheet' type='text/css'>
+<script>
+ (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+ (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+ m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+ })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+ ga('create', 'UA-77454349-1', 'auto');
```

Comment:
analytics is of David's account

On Mon, May 9, 2016 at 9:09 PM, Daniel Doubrovkine (dB.) @dblockdotorg <
notifications@github.com> wrote:

> In public/index.html.erb
> <https://github.com/dblock/slack-amber-alert/pull/4#discussion_r62520984>:
>
> > @@ -5,24 +5,137 @@
> >    <script src="/scripts/url.min.js"></script>
> >    <script src="/scripts/register.js"></script>
> >    <script src="/scripts/stats.js"></script>
> > +  <link rel="stylesheet" href="css/style.css">
> > +    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
> > +    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
> > +<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
> > +   <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
> > +   <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700,400italic,300italic,300,600italic,600,700italic,800,800italic' rel='stylesheet' type='text/css'>
> > +<script>
> > + (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
> > + (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
> > + m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
> > + })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
> > +
> > + ga('create', 'UA-77454349-1', 'auto');
>
> Who's GoogleAnalytics account is this? Just curious.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dblock/slack-amber-alert/pull/4/files/aa8fbe8ee293bcbdab7d716b43aca3354a9ac83c#r62520984>
>